src/drawing.cpp

```diff
@@ -439,18 +276,7 @@ void IRAM_ATTR DrawLoopTaskEntry(void *)
         if (wifiPixelsDrawn == 0)
             localPixelsDrawn = LocalDraw();
 
-        #if USE_MATRIX
-            // Swap the back buffer forward, set the auto-brightness.  It only does anything if pixels were drawn,
-            // but is always called for orthogonality with the PreDraw() call.
-            MatrixPostDraw(localPixelsDrawn + wifiPixelsDrawn);
-        #endif
-
-        #if USE_STRIP
-            if (wifiPixelsDrawn)
-                ShowStrip(wifiPixelsDrawn);
-            else if (localPixelsDrawn)
-                ShowStrip(localPixelsDrawn);
-        #endif
+        graphics->PostProcessFrame(wifiPixelsDrawn, localPixelsDrawn);

```

Comment:
So the Screen class also has a prepare/submit frame virtual function.  Not for this checkin, but I'd suggest unifying those since it inherits from GFXBase as well.  In other words, make Screen override Prepare/Post frame calls, and then in the one or two places in screen.h where the current begin/endframe, replace it woth Prepare/Post

Reply:
Ah, I didn't know that, thanks for the pointer! I'll look into that once on this PR CI has passed, I've tested it works on my devices, and merged it.